CMakeLists.txt

```diff
@@ -253,3 +256,14 @@ endif()
 
 install_obs_plugin_with_data(obs-browser data)
 install_obs_plugin(obs-browser-page)
+
+if(APPLE)
```

Comment:
This change must be reverted - CEF will make OBS crash if it is found and loaded without OBS being packaged into an application bundle

Adding this would prohibit debugging OBS in Xcode without bundling it manually on every build. Also this is taken care of by `full-build-macos` when an application bundle is created.

Reply:
Done